.travis.yml

```diff
@@ -36,7 +36,7 @@ install:
 script:
   # Only run code generation tests on OSX -- linux requires sudo to install OpenCV dependencies, and that environment
   # will not be able to run MainWindowTest.testDragOperationFromPaletteToPipeline
-  - ./gradlew check jacocoTestReport jacocoRootReport --stacktrace -Pheadless=true -PlogTests --scan;
```

Comment:
Why the removal of the `--scan`?

Reply:
It was causing Travis to fail from having to accept the license agreement. Since Travis is getting removed anyway, I'm not worried about temporarily reducing its workload to keep the important stuff working until then